CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
 - Fix APM network logging on Android ([#1253](https://github.com/Instabug/Instabug-React-Native/pull/1253)).
 - Fix an OOM (out-of-memory) crash while saving network logs on Android ([#1244](https://github.com/Instabug/Instabug-React-Native/pull/1244)).
 
+### Added
+
+- Enhance the network interceptor to capture some client error messages ([#1256](https://github.com/Instabug/Instabug-React-Native/pull/1256)).
+
```

Comment:
Can you please move the "Added" section before the "Fixed" section as described in the Changelog guidelines? 🙏🏼